composer.json

```diff
@@ -4,7 +4,7 @@
     "require": {
         "phing/phing": "~2.7",
         "jorgegc/phing-compass": "~1.0",
-        "jorgegc/phing-drush": "~4.0",
+        "jorgegc/phing-drush": "~4.2",
```

Comment:
There should be a `composer.lock` update as well https://github.com/DoghouseMedia/drupal-phing-build/blob/master/composer.lock#L394